### ISSUE TITLE
Notifications Not Stacking

### ISSUE THREAD

**Issue by yiruzu** [BeansFL/FL-Notify#2]
Notifications do not stack. They simply replace the previous notification instead of stacking on top or bottom.

**Comment by BeansFL** [comment]
This isnt an included feature yet.

**Comment by yiruzu** [comment]
> This isnt an included feature yet.

Yes ik this is a very important suggestion tho and it's kinda the only reason why I don't use it right now.

**Comment by BeansFL** [comment]
The thing is I wouldve fixed it but I sadly dont have any free time I can use for it due to my own server work. The frontend is opensource if you maybe wanna try it

**Comment by yiruzu** [comment]
> The thing is I wouldve fixed it but I sadly dont have any free time I can use for it due to my own server work. The frontend is opensource if you maybe wanna try it
> 
> 

I just don't know two shi*s about react but maybe imma try it out and if I get something working I'm gonna make a pull request

**Comment by BeansFL** [comment]
Thanks bro appreciated <3
